resource/decker/support/plugins/decker/decker.js

```diff
@@ -15,6 +15,10 @@ function onStart(deck) {
 
   deck.addEventListener("ready", () => {
     if (!printMode) {
+      totalSlides = deck.getTotalSlides();
+      console.log(
+        "[decker.js plugin] The deck has " + totalSlides + " total slides."
```

Comment:
remove this debug output in order to not fill up the console

Reply:
Done.